apps/meteor/client/components/message/Attachments/QuoteAttachment.tsx

```diff
@@ -25,42 +32,57 @@ const hover = css`
 	}
 `;
 
-export const QuoteAttachment: FC<MessageQuoteAttachment> = ({
-	author_icon: url,
-	author_name: name,
-	author_link: authorLink,
-	message_link: messageLink,
-	ts,
-	text,
-	attachments,
-}) => {
+type QuoteAttachmentProps = {
+	attachment: MessageQuoteAttachment;
+	message: IMessage;
+};
+
+export const QuoteAttachment = ({ attachment, message }: QuoteAttachmentProps): ReactElement => {
 	const format = useTimeAgo();
+
+	const msg: IMessage = {
+		_id: message._id,
+		ts: message.ts,
+		u: message.u,
+		_updatedAt: message._updatedAt,
+		rid: message.rid,
+		md: undefined,
+		msg: attachment.text,
+	};
+
+	const tokens = useParsedMessage(msg);
```

Comment:
You could do just `useParsedMessage({ ...msg, msg: attachment.text });`.

Also, you're passing a "fake" message here, since the text from the attachment is from another message. Should we perhaps create a hook to parse just text in this case?